concordium-node/src/grpc2.rs

```diff
@@ -575,6 +670,33 @@ pub mod types {
                     signatures.serial(&mut data);
                     Ok(data)
                 }
+                send_block_item_request::BlockItem::AccountTransactionV1(atv1) => {
+                    let patv1 = PreAccountTransactionV1 {
+                        header: atv1.header,
+                        payload: atv1.payload,
+                    };
+                    let (header, payload) = patv1.try_into()?;
+                    let signatures = atv1.signatures.require()?.try_into()?;
+                    let atv1 = concordium_base::transactions::AccountTransactionV1 {
+                        signatures,
+                        header,
+                        payload,
+                    };
+                    Ok(concordium_base::common::to_bytes(&Versioned::new(
+                        // TODO(drsk) is this the right version here?
+                        1.into(),
```

Comment:
I think it just needs to be
```suggestion
                        0.into(),
```

like the others